### ISSUE TITLE
Feature request: Enabling sequences of tasks with "id:" and "after:"

### ISSUE THREAD

**Issue by akusel** [mpcjanssen/simpletask-android#1223]
First, thanks a lot for this great app. I installed it a while ago, but just recently discovered its life-changing potential for myself.
I use Simpletask in conjunction with pter on my Linux machine. In addition to the actual todo.txt-syntax, pter supports sequences of tasks with the key:value pairs "id:xyz" and "after:xyz". With these, you can give a unique ID to one task and then refer to it in another task, making the second one show up only after the first has been completed. Example:
`Wash laundry due:2023-05-12 id:1`
`Dry laundry after:1`
If Simpletask could understand this syntax as well, this would make handling recurring sequences of small tasks a breeze. Also, planning bigger projects beforehand would become much easier. A big part of the weekly review could be automated.
